pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -1459,13 +1458,11 @@ const (
 
 // ExtendedClusterHealth stores health information of a cluster.
 type ExtendedClusterHealth struct {
-	Apiserver         HealthStatus `json:"apiserver,omitempty"`
-	Scheduler         HealthStatus `json:"scheduler,omitempty"`
-	Controller        HealthStatus `json:"controller,omitempty"`
-	MachineController HealthStatus `json:"machineController,omitempty"`
-	Etcd              HealthStatus `json:"etcd,omitempty"`
-	//  Deprecated: OpenVPN will be removed entirely in the future.
-	OpenVPN                      HealthStatus  `json:"openvpn,omitempty"`
```

Comment:
I'm not sure if this is breaking stuff and if we should postpone removing fields from the API until 2.27, when we're certain all components in a KKP system are on 2.26 already. Maybe it's overly cautious. Your call.

Reply:
Fair point. I did follow that policy for the `Cluster` spec field, so I should here as well.

WDYT, should the health status be always up or always down?